ci-operator/config/kubevirt/common-templates/kubevirt-common-templates-release-v0.31.yaml

```diff
@@ -0,0 +1,392 @@
+base_images:
+  base:
+    name: "4.17"
```

Comment:
These versions are newer than in `kubevirt-common-templates-master.yaml`.
Can you post another PR for `master` branch?

Reply:
Yes, I wanted to do it as follow up PR

Reply:
https://github.com/openshift/release/pull/57029